content/en/security_platform/application_security/getting_started/php.md

```diff
@@ -0,0 +1,84 @@
+---
+title: PHP Applications
+kind: documentation
+code_lang: php
+type: multi-code-lang
+code_lang_weight: 40
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-php'
+      tag: 'GitHub'
+      text: 'PHP Datadog Library source code'
+---
```

Comment:
The tracer and AppSec are separate in Github, so something like the following would be more appropriate:

```
    - link: 'https://github.com/DataDog/dd-appsec-php'
      tag: 'GitHub'
      text: 'PHP Datadog AppSec Library source code'
    - link: 'https://github.com/DataDog/dd-trace-php'
      tag: 'GitHub'
      text: 'PHP Datadog Tracer Library source code'
```